python/django_bolt/serializers/decorators.py

```
@@ -251,7 +285,11 @@ def collect_field_validators(cls: type[Serializer]) -> dict[str, list[FieldValid
         if not hasattr(base, "__dict__"):
             continue
 
-        for _name, value in base.__dict__.items():
+        for _name, raw in base.__dict__.items():
```

Ruff (0.16.1)

[warning] When using only the values of a dict use the `values()` method

Replace `.items()` with `.values()`

(PERF102)